events-generator/src/main/java/org/moditect/jfrunit/generator/events/model/Attribute.java

```diff
@@ -0,0 +1,104 @@
+package org.moditect.jfrunit.generator.events.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Attribute {
+    static Map<String, String> attributeObjToJfrUnitType = new HashMap<>();
+
+    static {
+        attributeObjToJfrUnitType.put("long", "Long");
+        attributeObjToJfrUnitType.put("int", "Integer");
+        attributeObjToJfrUnitType.put("Thread", "org.moditect.jfrunit.ExpectedThread");
+        attributeObjToJfrUnitType.put("String", "java.lang.String");
+        attributeObjToJfrUnitType.put("Class", "org.moditect.jfrunit.ExpectedClass");
+        attributeObjToJfrUnitType.put("ClassLoader", "org.moditect.jfrunit.ExpectedClassLoader");
+        attributeObjToJfrUnitType.put("Module", "org.moditect.jfrunit.events.model.Module");
+        attributeObjToJfrUnitType.put("Package", "org.moditect.jfrunit.events.model.Package");
+        attributeObjToJfrUnitType.put("StackFrame", "org.moditect.jfrunit.ExpectedStackFrame");
+        attributeObjToJfrUnitType.put("StackTrace", "org.moditect.jfrunit.ExpectedStackTrace");
+        attributeObjToJfrUnitType.put("Method", "org.moditect.jfrunit.ExpectedMethod");
+        attributeObjToJfrUnitType.put("boolean", "Boolean");
+        attributeObjToJfrUnitType.put("byte", "Byte");
+        attributeObjToJfrUnitType.put("char", "Character");
+        attributeObjToJfrUnitType.put("double", "Double");
+        attributeObjToJfrUnitType.put("float", "Float");
+        attributeObjToJfrUnitType.put("short", "Short");
+
+        attributeObjToJfrUnitType.put("Timespan", "java.time.Duration");
+        attributeObjToJfrUnitType.put("Timestamp", "java.time.Instant");
```

Comment:
Let's create a temp map first and wrap that into an unmodifiable one, so that the exposed one is immutable.